kubernetes-docs/sc.md

```diff
@@ -0,0 +1,167 @@
+vsphere-volume-sc-slow.yaml
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: fast
+provisioner: kubernetes.io/vsphere-volume
+parameters:
+    diskformat: zeroedthick
+    datastore: VSANDatastore
+If datastore is member of DataStore Cluster or within some sub folder, the datastore folder path needs to be provided in the datastore parameter as below.
+parameters:
+   datastore:	DatastoreCluster/VSANDatastore
+```
+
+**Create the Storageclass**
+
+```
+$ kubectl create -f examples/volumes/vsphere/vsphere-volume-sc-fast.yaml
+```
+
+**Verifying storage class is created**
+
+```
+$ kubectl describe storageclass fast 
+Name:           fast
+IsDefaultClass: No
+Annotations:    <none>
+Provisioner:    kubernetes.io/vsphere-volume
+Parameters:     diskformat=zeroedthick,fstype=ext3
+No events.
+Create Persistent Volume Claim.
+Vsphere-volume-pvcsc.yaml
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: pvcsc001
+  annotations:
+    volume.beta.kubernetes.io/storage-class: fast
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 2Gi
+```
+
+**Create the persistent volume claim**
+
+```
+$ kubectl create -f examples/volumes/vsphere/vsphere-volume-pvcsc.yaml
+```
+
+**Verify persistent volume claim is created**
+
+```
+$ kubectl describe pvc pvcsc001
+Name:           pvcsc001
+Namespace:      default
+StorageClass:   fast
+Status:         Bound
+Volume:         pvc-83295256-f8e0-11e6-8263-005056b2349c
+Labels:         <none>
+Capacity:       2Gi
+Access Modes:   RWO
+Events:
+  FirstSeen     LastSeen        Count   From                            SubObjectPath   Type            Reason                  Message
+  ---------     --------        -----   ----                            -------------   --------        ------                  -------
+  1m            1m              1       persistentvolume-controller                     Normal          ProvisioningSucceeded   Successfully provisioned volume pvc-83295256-f8e0-11e6-8263-005056b2349c using kubernetes.io/vsphere-volume
+```
+
+Persistent Volume is automatically created and is bounded to this pvc.
+
+**Verify persistent volume claim is created**
+
+```
+$ kubectl describe pv pvc-83295256-f8e0-11e6-8263-005056b2349c
+Name:           pvc-83295256-f8e0-11e6-8263-005056b2349c
+Labels:         <none>
+StorageClass:   fast
+Status:         Bound
+Claim:          default/pvcsc001
+Reclaim Policy: Delete
+Access Modes:   RWO
+Capacity:       2Gi
+Message:
+Source:
+    Type:       vSphereVolume (a Persistent Disk resource in vSphere)
+    VolumePath: [datastore1] kubevols/kubernetes-dynamic-pvc-83295256-f8e0-11e6-8263-005056b2349c.vmdk
+    FSType:     ext3
+No events.
+Note: VMDK is created inside kubevols folder in datastore which is mentioned in 'vsphere' cloudprovider configuration. The cloudprovider config is created during setup of Kubernetes cluster on vSphere.
+Create Pod which uses Persistent Volume Claim with storage class.
```

Comment:
This should be a separate section - "Create Pod which uses Persistent Volume Claim with storage class". This problem seems consistent across all the pages.